src/IECoreHoudini/FromHoudiniGeometryConverter.cpp

```diff
@@ -959,6 +959,35 @@ DataPtr FromHoudiniGeometryConverter::extractStringData( const GU_Detail *geo, c
 	return new StringData();
 }
 
+
+bool FromHoudiniGeometryConverter::hasOnlyOpenPolygons( const GU_Detail *geo )
```

Comment:
Does this approach work for a single SOP containing both open and closed polygons, but using different names?

Reply:
Discussed offline. The situation I was describing is already hosed on master, so we can merge this as an improvement and fix that up later on. I'm quite surprised production hasn't noticed (or prioritized) this limitation yet.